docs/user_guide.rst

```diff
@@ -18,57 +18,98 @@ Configuration
 -------------
 
 *tox* needs a configuration file where you define what tools you need to run and how to provision a test environment for
-these. The canonical file for this is the ``tox.ini`` file. For example:
-
-.. code-block:: ini
-
-    [tox]
-    requires =
-        tox>=4
-    env_list = lint, type, py{38,39,310,311}
-
-    [testenv]
-    description = run unit tests
-    deps =
-        pytest>=7
-        pytest-sugar
-    commands =
-        pytest {posargs:tests}
-
-    [testenv:lint]
-    description = run linters
-    skip_install = true
-    deps =
-        black==22.12
-    commands = black {posargs:.}
-
-    [testenv:type]
-    description = run type checks
-    deps =
-        mypy>=0.991
-    commands =
-        mypy {posargs:src tests}
+these. The canonical file for this is either a ``tox.toml`` or ``tox.ini`` file. For example:
+
+    .. tab:: TOML
+
+       .. code-block:: toml
+
+            requires = ["tox>=4"]
+            env_list = ["lint", "type", "3.13", "3.12", "3.11"]
+
+            [env_run_base]
+            description = "run unit tests"
+            deps = [
+                "pytest>=8",
+                "pytest-sugar"
+            ]
+            commands = [["pytest", { replace = "posargs", default = ["tests"] }]]
+
+            [env.lint]
+            description = "run linters"
+            skip_install = true
+            deps = ["black"]
+            commands = [["black", { replace = "posargs", default = ["."]} ]]
+
+            [env.type]
+            description = "run type checks"
+            deps = ["mypy"]
+            commands = [["mypy", { replace = "posargs", default = ["src", "tests"]} ]]
+
+
+    .. tab:: INI
+
+       .. code-block:: ini
+
+           [tox]
+            requires =
+                tox>=4
+            env_list = lint, type, 3.{3,2,1}
```

Comment:
```
env_list = lint, type, 3.{13,12,11}
```
To make it correspond to the TOML version.

Reply:
Ah good typo spot.